Samiksha/html/Assignment1/Assignment no 1.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <title>this is my title</title>
+</head>
+  <body>
+   <h1>
+      HTML
+   </h1>
+    <h1>
+      Introduction
+    </h1>
+<<<<<<< HEAD
+    <p>
+      HTML is a standard markup language for web pages.
+    </p>
+    <h1> HTML Editor </h1>
+    <h5> Notepad or TextEdit </h5>
+    <p> 
+      Web Pages can be created and modified by using proffessional HTML editors. <br> 
+      However, for learning HTML we recommend a simple text editor like Notepad(PC) or        TextEdit(Mac).
+    </p>
+   <h1>
+      HTML
+    </h1>
+    <h1>
+      Introduction
+    </h1>
+    <p>
+      HTML is a standard markup language for web pages.
+    </p>
+    <hr size="1" width="100%" color="black">
+    <h1> HTML Editor </h1>
+    <h5> Notepad or TextEdit </h5>
+    <p> 
+      Web Pages can be created and modified by using proffessional HTML editors. <br> 
+      However, for learning HTML we recommend a simple text editor like Notepad(PC) or        TextEdit(Mac).
+=======
+     <p>
+      HTML is a standard markup language for web pages.
+    </p>
+      <h1> HTML Editor </h1>
+      <h5> Notepad or TextEdit </h5>
+      <p> Web Pages can be created and modified by using proffessional HTML editors. <br> 
+          However, for learning HTML we recommend a simple text editor like Notepad(PC) or TextEdit(Mac).
+    </p>
+   <h1> HTML </h1>
+   <h1> Introduction </h1>
+    <p>
+      HTML is a standard markup language for web pages.
+    </p>
+    <hr size="1" width="100% color="black">
+    <h1> HTML Editor </h1>
+    <h5> Notepad or TextEdit </h5>
+    <p> Web Pages can be created and modified by using proffessional HTML editors. <br> 
+    However, for learning HTML we recommend a simple text editor like Notepad(PC) or TextEdit(Mac).
+>>>>>>> 84232bb5105d425924556eabcefbf339be0ceb5b
```

Comment:
unnecessary line `>>>>>>> 84232bb5105d425924556eabcefbf339be0ceb5b` in other codes too